src/types/Entity.ts

```diff
@@ -104,7 +104,10 @@ export interface Entity extends WikiEntity {
 
 export type Image = {
   url: string;
-  alt: string;
+  alt?: string;
+  sourceUrl?: string;
+  imageDb?: boolean;
+  urlByType?: string[];
```

Comment:
hmmm, you have used it as an object here `currentThumbnail.urlByType?.[settings.imageType]`